sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSortExec.scala

```diff
@@ -87,10 +88,11 @@ case class GpuSortExec(
     gpuSortOrder: Seq[SortOrder],
     global: Boolean,
     child: SparkPlan,
-    sortType: SortExecType)(cpuSortOrder: Seq[SortOrder])
+    sortType: SortExecType)(
+    cpuSortOrder: Seq[SortOrder], writeTrackers: Option[Seq[GpuWriteJobStatsTracker]] = None)
   extends ShimUnaryExecNode with GpuExec {
```

Comment:
Nit: Add comments like: 
For Spark before 340, GPU write exec will perform some sort operation internally before writing,
here pass in write stat tracker to update the sort metric in write exec.